frontend/public/vendor.scss

```diff
@@ -42,14 +42,14 @@
 @import '~@patternfly/react-catalog-view-extension/dist/sass/properties-side-panel';
 @import '~@patternfly/react-catalog-view-extension/dist/sass/vertical-tabs';
 @import '~@patternfly/patternfly/sass-utilities/all';
-@import "~@patternfly/patternfly/variables";
-@import "~@patternfly/patternfly/fonts";
-@import "~@patternfly/patternfly/base";
+@import "~@patternfly/patternfly/base/variables";
```

Comment:
Please re-alphabetize the imports to keep them in order.

Reply:
fixed